buildSrc/src/main/kotlin/com/datadog/gradle/plugin/benchmark/ReviewBenchmarkResultsTask.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2020 Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.benchmark
+
+import java.io.File
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.TaskAction
+
+open class ReviewBenchmarkResultsTask : DefaultTask() {
+
+    internal lateinit var extension: ReviewBenchmarkExtension
+    internal lateinit var buildDir: File
+
+    private val visitor = BenchmarkJsonFileVisitor()
+
+    init {
+        group = "datadog"
+        description = "Review the benchmark results and ensure they are below the specified threshold"
+    }
+
+    // region Task
+
+    @TaskAction
+    fun applyTask() {
+        val targetDir = getInputDir()
+        targetDir.listFiles()?.forEach {
```

Comment:
maybe you can do an extra check here to make sure the directory exists before listing the files.